### PR TITLE
[FIND] Update Romanian (ro-RO) translation

### PR DESCRIPTION
**Pull request by AndreiMiloiu**
Attendum to PR https://github.com/reactos/reactos/pull/5908 and I fixed a typo.

### CODE REVIEW

**Comment by AndreiMiloiu** [comment]
Cc @reactos/lang-romanian

**Review by JoachimHenze — APPROVED** [review]
Approved. Length of the line is 78 chars, so the console line length of 80 should not overflow.

File: base/applications/cmdutils/find/lang/ro-RO.rc [hunk 1]
```diff
@@ -16,8 +16,7 @@ FIND [/V] [/C] [/N] [/I] [/OFF[LINE]] ""șir"" [fișier...]\n\
   /C         Numără liniile ce conțin șirul.\n\
   /N         Numără rândurile afișate, începând cu 1.\n\
   /I         Ignoră diferențele între majuscule și minuscule.\n\
-  /OFF[LINE] Nu omite fișierele care care au configurate atributul\n\
-             offline (deconectat)."
+  /OFF[LINE] Nu omite fișierele care au setate atributul offline (deconectat)."
```
Comment (GeoB99) [inline base/applications/cmdutils/find/lang/ro-RO.rc:19]: Could you please show a screenshot of how the `FIND` command line utility looks before and after this change? Personally I'm not that keen of this change so I want to see the actual benefits at hand.

Comment (AndreiMiloiu) [reply base/applications/cmdutils/find/lang/ro-RO.rc:19]: I can try to do a screenshot if you want. The benefit is that I fixed a typo. The word "care" was repeated two times. By removing it once the number of characters was reduced.

Comment (AndreiMiloiu) [reply base/applications/cmdutils/find/lang/ro-RO.rc:19]: Before

find command before (test)


After

find command after (test)

Comment (AndreiMiloiu) [reply base/applications/cmdutils/find/lang/ro-RO.rc:19]: I think I understand what you mean. I will let the description on two rows as before.

Comment (GeoB99) [reply base/applications/cmdutils/find/lang/ro-RO.rc:19]: >The benefit is that I fixed a typo. The word "care" was repeated two times. By removing it once the number of characters was reduced.

Ah sorry I haven't seen this. Yeah this is good. 👍

Comment (AndreiMiloiu) [reply base/applications/cmdutils/find/lang/ro-RO.rc]: So, should be the text again on two rows or only on one?

Comment (GeoB99) [reply base/applications/cmdutils/find/lang/ro-RO.rc]: Two rows.

Comment (AndreiMiloiu) [reply base/applications/cmdutils/find/lang/ro-RO.rc]: I see, thank you!

**Pull request merged**